api/document/js/Api.ts

```diff
@@ -14,6 +14,8 @@ export function setStatusError(e: Error) {
   const data = e['response'] && e['response']['data'];
   if (data) {
     errMsg += '\n' + makeErrorFriendly(data);
+  } else {
+    console.error(e);
```

Comment:
Rather than logging it here, we could instead log the error in `create-editor-state.ts`'s `catch` clause, because that's currently the only place we're calling `setStatusError` without re-throwing the error.  I'm not sure which strategy is best, though.

Reply:
Also, I'm not sure if we want to be extra careful and test to see if `window.console.error` even exists before attempting to call it.  Do all browsers have that function these days?

Reply:
MDN claims support back to IE8, so I think we're good. Thanks for thinking of it, though!